week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -20,59 +20,77 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 //1)Create the function inside, Think about nested loops
+
+let arr = [1, 45, 43, "Hello", "Goodbye"]
+let newArr = []
 function sortAges(arr) {
-  //create function here
+  
+  for (let i = 0; i < arr.length; i++) {
+    
+    if (typeof arr[i] === 'number') {
+      newArr.push(arr[i])
+    }

```

Comment:
So far, it looks good.
If you wouldn't like to write your own sorting algorithm then you can go for the simple way by using the built-in `sort()` method on array.